crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -0,0 +1,628 @@
+
+//
+// THIS FILE IS GENERATED FORM neon.spec DO NOT CHANGE IT MANUALLY
```

Comment:
Without the generator this comment is spurious, and the contents of this file should be inlined within `neon.rs`.

Reply:
That will make adding more intrinsics harder.

Reply:
We can revisit re-adding this once a generator is added to the project. Without a generator in tree it is hard to justify splitting the intrinsics into two files. 

If you only want to use the generator locally, then you can just copy paste its output somewhere else. Do you have other difficulties in mind?

Reply:
I split out the generator in a own project: https://github.com/Licenser/neongen

I get the reservations about a generator in the stdlib as well as this is the right way to generate them at least as long as it's in std and 'the de facto standard' by that. This generator works great for me, it's fine if it doesn't for others. So it can grow on it's own now and evolve without causing change in a stdlib - wich makes a lot of sense i think! It's much easier to iterate / improve in isolation.

Ultimately it shouldn't matter if the functions are artisanal crafted with love, written by a mad AI that escaped it's cage or a generator (as long as they work, and fulfil the requirements - has a test and validates the intrinsic).

I hope to add more intrinsics over time, probably in batches of a few dozen/hundred at a time to a) make review easier and b) take the pressure off to add ten million pages (slight hyperbole) pages of intrinsics which seems to be a insurmountable task. Having it all in one file means for every new batch it got to be fiddled into the "right" place in the neon.rs and the tests and exports too. In addition to that I'm not sure it's sensible for all intrinsics to be generated, some seem to require some gymnastics that will be very hard to generalise (or generalising them would be more work then just typing them down) so when all is done (famous last words) I suspect a 90/10 split of generated and manual intrinsics for arm and the split in two files would still make sense.

Reply:
> I get the reservations about a generator in the stdlib as well as this is the right way to generate them at least as long as it's in std and 'the de facto standard' by that. 

Putting a generator into this project doesn't add it to the stdlib. Only the contents of the `core_arch` crate are actually added to the standard library.

Reply:
The other points you mention make sense. I still think it would be helpful to have such a generator here in tree somewhere (e.g. in `crates/stdarch-gen` or similar, just like we do for `stdarch-verify`), so that it can be useful to others.

Reply:
That's a good idea! I'll open a new PR for that, and we can grow that on it's own without having to cross the discussions of how what gets generated and intrinsics and how they work/or don't, sounds good?

Reply:
Yes, that's sound like a better plan.

Reply:
See https://github.com/rust-lang/stdarch/pull/802